CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 Changes
 =======
 
+# 0.12.0 / Unreleased
+
+**This version drops the support of Chef < 12.7**
+
+* [FEATURE] Add support for Chef 13 and 14. [#101][] [#104][] [@martinisoft][]
```

Comment:
* did we not support Chef 13 or 14 in the past?
* do we support Chef 15 now?

Reply:
Fair enough. I've reformulated for better clarity.